src/app/pages/test-centre-journal/test-centre-journal.page.html

```diff
@@ -32,7 +32,9 @@ <h6 class="last-sync">Last sync: {{pageState.lastRefreshedTime$ | async}}</h6>
                     <candidate-search-card [testCentreResults]="testCentreResults"></candidate-search-card>
                 </tab>
                 <tab [tabTitle]="'View Journals'">
-                    <view-journals-card [testCentreResults]="testCentreResults"></view-journals-card>
+                    <view-journals-card
```

Comment:
is above comment is addressed this can be un-done

Reply:
removed